### PR TITLE
Impl: Faq hi

### PR DESCRIPTION
**Pull request by kozr**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

"hardcode" cmd-f faq section

Fixes #29 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Works on staging site

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kozr** [comment]
Ugh, I hate doing faq...

**Comment by kozr** [comment]
I kind of stopped thinking about monorepo configs because time was tight.

**Review by rmehri01 — COMMENTED** [review]
This is definitely one of the harder parts but it looks really nice! Just a small thing is that it's slightly overlapping for me on mobile:

File: src/components/faqTemplates/shared/FaqList.jsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
-      <div key={question}>
-        <ExpandableFaq question={question} answer={answer} />
-      </div>
-    )) ?? null
+    <FaqList>
+      {list?.map(({ question, answer }) => (
+        <div key={question}>
+          <ExpandableFaq question={question} answer={answer} decor={decor} />
+        </div>
+      )) ?? null}
```
Comment (yehee) [inline src/components/faqTemplates/shared/FaqList.jsx:16]: Never seen this syntax 👀  good to know!

File: src/pages/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import GlobalStyles from '@styles/global'
 import fireDb from '@utilities/firebase'
 import { serialize } from '@utilities/format'
-import FAQExpandable from '@components/faqTemplates/ExpandableWithCategories'
+import FAQExpandable from '@components/faqTemplates/Cmdf2021'
```
Comment (yehee) [inline src/pages/index.js:6]: Any reason for this change?

**Review by yehee — COMMENTED** [review]
FAQ section definitely takes up a lot of time! Also spotted a few overlapping, included here for your reference. Let me know if you need help with anything!
Screen Shot 2021-02-10 at 1 20 43 AM
Screen Shot 2021-02-10 at 1 20 24 AM

File: src/pages/index.js (hunk 2 above)
Comment (kozr) [reply src/pages/index.js:6]: Realized FAQ may not be one of the sections that we can just use a template for. There are too many custom css styling. Might create a node module next year instead. idk :)

**Review by rmehri01 — COMMENTED** [review]
I think based on the figma this is unintended?

**Comment by rmehri01** [comment]
@kozr I'm not sure if it looks the same for you but on my phone and in browser it looks like this to me:

Screen Shot 2021-02-13 at 5 13 44 PM

I tried doing `width: auto` instead and I think it is better?

Screen Shot 2021-02-13 at 5 16 47 PM

**Comment by yehee** [comment]
@kozr Still seeing some overlappings and weird margin issue with FAQ section 🤔  Also seems like changing `background-size: contain` to `cover` may have affected About section, would you be able to take a look?
Screen Shot 2021-02-13 at 4 54 47 PM

**Review by rmehri01 — APPROVED** [review]
Looks good for me now!

**Pull request merged**